subsys/secure_storage/Kconfig

```diff
@@ -65,7 +65,7 @@ endif
 
 menuconfig SECURE_STORAGE_ITS_STORE_MODULE
 	bool "ITS store module"
-	imply FLASH # for FLASH_HAS_DRIVER_ENABLED
+	depends on FLASH # for FLASH_HAS_DRIVER_ENABLED
```

Comment:
This was originally added to be able to check `FLASH_HAS_DRIVER_ENABLED` in `Kconfig.its_store`.
This line can now be removed completely as it's expected that `FLASH` will be enabled elsewhere. `FLASH_HAS_DRIVER_ENABLED` will still be checked in `Kconfig.its_store` (and `FLASH` is a prerequisite of it).